fractions/javaee/undertow/module.conf

```diff
@@ -3,3 +3,4 @@ org.wildfly.extension.undertow
 io.undertow.core
 io.undertow.servlet
 org.ow2.asm
+org.wildfly.swarm.security
```

Comment:
I'll try again, but my understanding is that without the module dependency, the `SecurityFraction` class won't be visible to the customizer.